src/NServiceBus.Core.Tests/Msmq/MsmqQueueCreatorTests.cs

```diff
@@ -180,34 +187,27 @@ public void Should_not_add_everyone_and_anonymous_to_already_existing_queues()
             Assert.False(queue.TryGetPermissions(LocalEveryoneGroupName, out nullBecauseRevoked, out accessControlEntryType));
             Assert.False(queue.TryGetPermissions(LocalAnonymousLogonName, out nullBecauseRevoked, out accessControlEntryType));
             Assert.IsNull(accessControlEntryType);
+
         }
 
         [Test]
         public void Should_allow_queue_names_above_the_limit_for_set_permission()
```

Comment:
wasn't the scenario here that the queue was somehow already created by the user and therefore we try to use it although we wouldn't be able to create it? (in that case, I guess it makes sense)